hocs/withI18n.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * Copyright (c) 2017-present, Global Digital Library.
+ * 
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ * 
+ */
+
+// @flow
```

Comment:
Er dette utkommentert kode, eller en kommentar jeg ikke forstår? 
Uansett, vi trenger vel ingen av delene 😉

Reply:
Det er kommentaren som forteller at vi har typet filen med flow. https://flow.org/en/docs/usage/#toc-prepare-your-code-for-flow